apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Convert2.java

```diff
@@ -42,7 +42,7 @@ public static String defaultRsAccount(long accountId) {
 
 
     public static long fromEpochTime(int epochTime) {
-        return epochTime * 1000L + Genesis.EPOCH_BEGINNING - 500L;
+        return epochTime * 1000L + GenesisImporter.EPOCH_BEGINNING - 500L;
```

Comment:
better to make it non static

Reply:
No sense to do that here. Convert2 is not CDI.